packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import ButtonGroup from 'terra-button-group';
+import 'terra-base/lib/baseStyles';
+import Menu from './_CollapsibleMenu';
+import styles from './CollapsibleMenuView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Sets the item's text
+   */
+  text: PropTypes.string.isRequired,
+
+  /**
+   * Bounding container for the menu, will use window if no value provided.
+   */
+  boundingRef: PropTypes.func,
+
+  /**
+  * An optional icon. Nested inline with the text when provided
+  */
+  icon: PropTypes.element,
+
+  /**
+   * Indicates if only the icon should display when item is face-up. (Text should still be given to be displayed when item is in the menu)
+   */
+  isIconOnly: PropTypes.bool,
+
+  /**
+  * Reverses the position of the icon and text
+  */
+  isReversed: PropTypes.bool,
+
+  /**
+   * Indicates if the item start as selected when placed in a button group
+   */
+  isSelected: PropTypes.bool,
+
+  /**
+   * Indicates that clicking on this item while displayed inside a menu should close the menu
+   */
+  shouldCloseOnClick: PropTypes.bool,
+
+  /**
+   * List of Menu.Items to display in a submenu when this item is selected.
+   */
+  subMenuItems: PropTypes.arrayOf(PropTypes.element),
+
+  /**
+   * Callback function for when the item is clicked
+   */
+  onClick: PropTypes.func,
+
+  /**
+   * Width for the menu
+   */
+  menuWidth: PropTypes.oneOf(['160', '240', '320', '640', '960', '1280', '1760']),
+};
+
+const contextTypes = {
+  isCollapsibleGroupItem: PropTypes.bool,
+  isCollapsibleMenuItem: PropTypes.bool,
+};
+
+const defaultProps = {
+  isSelected: false,
+  isReversed: false,
+  text: '',
+  shouldCloseOnClick: true,
```

Comment:
Do we want the default boolean value to be true? Should `_CollapseMenu` handle this?